### ISSUE TITLE
Consider adjusting UI layout of variable filters, making it more obvious

### ISSUE THREAD

**Issue by NickJosevski** [OctopusDeploy/Issues#4190]
> ... the Tenant tag set filter is not under the "Scope" section so it's not very intuitive that specifying a tag would filter based on the scope.

Reported here: https://secure.helpscout.net/conversation/504424506?folderId=1228416

**Comment by nkojuharov** [comment]
Just to clarify the issue here.  Per your comments "The focus for the All Variables screen is to filter what will apply at deployment time to the tenants that match the tags specified in the filter".

The problem I see is that when filtering based on the Tenant Tags, the Tenant Preview shows the right set of tenants, but the All Variables screen still shows the variables for tenants that don't match the tag filter (and aren't displayed in the Tenant Preview).  

So, basically, the Tenant Tags filter only works if you explicitly defined the tag in the variable scope.  If the variable scope has no explicit tag but the variable is assigned to a tenant with a tag, (thus, the tag scope is implicitly defined through the tenant) the filter doesn't work as expected.
